### PR TITLE
Update cookbook.rst

### PR DESCRIPTION
**Pull request by HarshMN2345**
This pull request introduces improvements to the project's documentation. It refactors the existing content in the Cookbook section to make it more readable and understandable. The information is now presented in a structured format with clear instructions on how to change API request timeouts and take control over the session of requests.

### CODE REVIEW

**Comment by HarshMN2345** [comment]
@ryshu please review this

**Comment by ryshu** [comment]
Apart from the fact that we don't normally close MRs to be able to follow the evolution of changes, it seems good to me.

I'm asking for another reread, however, because I wrote the documentation so I don't have a critical view on whether or not the content of the latter has been improved.

**Comment by HarshMN2345** [comment]
Previous one was difficult to understand

**Comment by HarshMN2345** [comment]
@nils-van-zuijlen Please review this

File: docs/guides/cookbook.rst [hunk 1]
```diff
@@ -1,43 +1,54 @@
...
-  You can define it in seconds and it will overloads all timeouts in this package.
-* Via the API constructor directly, passing :attr:`~novu.api.base.Api.requests_timeout` in keywords arguments during the initialisation.
+1. **Using an Environment Variable**
 
-.. code-block:: python
+   You can set an environment variable named ``NOVU_PYTHON_REQUESTS_TIMEOUT`` to specify the timeout in seconds. This variable will override all timeouts in this package.
+
+   Example:
+
+   .. code-block:: python
```
Comment (nils-van-zuijlen) [inline docs/guides/cookbook.rst:15]: You can use all of the Pygments language codes, bash seems appropriate here.

```suggestion
   .. code-block:: bash
```

**Review by nils-van-zuijlen — APPROVED** [review]
Yes, the changes make the documentation clearer, and easier to follow. Thank you!

**Comment by HarshMN2345** [comment]
@nils-van-zuijlen is it good to go for merging?

**Pull request merged**